networking_calico/plugins/ml2/drivers/calico/mech_calico.py

```diff
@@ -773,14 +773,19 @@ def update_port_postcommit(self, context):
 
             # Now, fork execution based on the type of update we're performing.
             # There are a few:
+            # - a pre live-migration notice (binding profile has a migrating_to
+            #   key with the future nova-compute host as the value), which we
+            #   do nothing with right now.
             # - a port becoming bound (binding vif_type from unbound to bound);
             # - a port becoming unbound (binding vif_type from bound to
             #   unbound);
             # - an Icehouse migration (binding host id changed and port bound);
             # - an update (port bound at all times);
             # - a change to an unbound port (which we don't care about, because
             #   we do nothing with unbound ports).
-            if port_bound(port) and not port_bound(original):
+            if port['binding:profile'].get('migrating_to') is not None:
```

Comment:
Please can you add a `'binding:profile' in port` check here, to allow for older OpenStacks that don't have that key.  (And also our UTs!)

Reply:
No problem -- done.  Did a`d.get('key', {})` to keep it all on one line.